sample/src/TonaApp/Util.hs

```diff
@@ -0,0 +1,32 @@
+module TonaApp.Util where
```

Comment:
I feel this module name is not bad, but not good. It is because the name "Util" is too general. The name does not provide any information, is not self-explanatory. Further, this module is actually the module for a specific purpose: to handle JST.

My proposion is as follows:

1. Rename the module name to `TonaApp.JST`
2. Rename the `getCurrentDayJST` to `getCurrentDay`
3. Rename the `midnightTimeJST` to `midnight`
4. Change the module to export only `getCurrentDay` and `midnightTime`.


Then import it in the `Main.hs` as follows:

```haskell
import qualified TonaApp.JST as JST
```

Now you can call `JST.getCurrentDay` and `JST.midnight`, which appearently similar to _class methods_ of the OOP.